infra/core/managed-cluster/aks-cluster.parameters.json

```diff
@@ -84,9 +84,26 @@
           "kustomizations": {
             "timeoutInSeconds": 600,
             "syncIntervalInSeconds": 600,
-            "clusterPath": "./clusters/#{{ lower(environment) }}/01",
-            "infraPath": "./infra/#{{ lower(environment) }}/01",
-            "servicesPath": "./services/#{{ lower(environment) }}/01"
+            "clusterPath": "./clusters/#{{ lower(environment) }}/0#{{ environmentId }}",
+            "infraPath": "./infra/#{{ lower(environment) }}/0#{{ environmentId }}",
+            "servicesPath": "./services/#{{ lower(environment) }}/0#{{ environmentId }}",
+            "postBuild": {
+              "substitute": {
+                "ENVIRONMENT": "#{{ lower(environment) }}",
+                "ACR_NAME": "#{{ infraResourceNamePrefix }}#{{ nc_resource_containerregistry }}#{{ nc_instance_regionid }}01",
+                "CLUSTER": "0#{{ environmentId }}",
+                "SERVICEBUS_RG": "#{{ servicesResourceGroup }}",
+                "SERVICEBUS_NS": "#{{ infraResourceNamePrefix }}#{{ nc_resource_servicebus }}#{{ nc_instance_regionid }}01",
+                "POSTGRES_SERVER_RG": "#{{ dbsResourceGroup }}",
+                "POSTGRES_SERVER": "#{{ dbsResourceNamePrefix }}#{{ nc_resource_postgresql }}#{{ nc_instance_regionid }}01",
+                "INFRA_RG": "#{{ servicesResourceGroup }}",
+                "TEAM_MI_NAME": "#{{ infraResourceNamePrefix }}#{{ nc_resource_managedidentity }}#{{ nc_instance_regionid }}01",
+                "CLUSTER_OIDC": "",
+                "TENANT_ID": "#{{ subscriptionName }}",
+                "SUBSCRIPTION_ID": "#{{ subscriptionId }}",
+                "SUBSCRIPTION_NAME": "#{{ tenantId }}"
```

Comment:
should this be tenantiD?? for sub name??

Reply:
you got tenantID and sub name wrong way round

Reply:
Good spot. I have updated it.